SPECS/libvirt/libvirt.spec

```diff
@@ -688,7 +688,7 @@ Requires: gzip
 Requires: bzip2
 Requires: lzop
 Requires: xz
-Requires: swtpm-tools
+#Requires: swtpm-tools
```

Comment:
Could you also provide documentation as to why swtpm-tools is not required anymore and why this change is being made.

Please add it to the changelog for this current spec file since you are making a change here

Reply:
this was temporarily commented out for me the trigger libvirt build, will revert back.